src/main.rs

```diff
@@ -142,6 +145,14 @@ fn protected_main() {
     #[cfg(target_os = "windows")]
     windows_attach_to_console();
 
+    match Config::load_from_path(&config_path()) {
+        Ok(config) => config.write_to_env(),
+        Err(err) => {
+            eprintln!("{err}");
+            error!("{err}");
```

Comment:
Do we need them both?

Reply:
I don't remember my exact reasoning, but now I realize that logger is not set up at this point anyway, so `error!` is useless. I'll delete it then.
Thanks for pointing that out.